### PR TITLE
GHO-13: further reading part 2

### PR DESCRIPTION
**Pull request by rupl**
# GHO-13

I closed #34 in favor of this PR. It's now a field with the `link_attributes` thing configured.. but I don't know how to print the information as an independent tag. It renders as a `source` attribute on the `<a>` tag itself... exactly as the module describes. I spent my afternoon trying to extract that link and print it independently, but could not succeed. If you have any ideas let me know. If not, then we have to close this PR and move onto the Paragraphs option.

At a point of desperation, I tried rendering it as a pseudo but the browser is smarter than me and wouldn't let me remove the underline of the pseudo element (makes sense in terms of making deceptive elements that are clickable but look otherwise)

```css
.gho-further-reading a[source]::after {
  content: attr(source);
  text-decoration: none; /* this does nothing, so our design cannot be achieved */
}
```

### CODE REVIEW

File: html/themes/custom/common_design_subtheme/templates/fields/field--field-further-reading.html.twig [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+{% else %}
+  <div{{ attributes.addClass(classes) }}>
+    <h3{{ title_attributes.addClass(title_classes) }}>{{ label }}</h3>
+    {% if multiple %}
+      <div class="field__items">
+    {% endif %}
+    {% for item in items %}
+      <div{{ item.attributes.addClass('field__item') }}>
+        {{ item.content }}
+        <span class="source">{# 💰 #}</span>
```
Comment (rupl) [inline html/themes/custom/common_design_subtheme/templates/fields/field--field-further-reading.html.twig:79]: here's the prize at the end of the 🌈

**Review by orakili — COMMENTED** [review]
I created a follow-up PR: https://github.com/UN-OCHA/gho-site/pull/38 to add a custom widget and formatter to handle the source field. The PR is currently against this branch.

**Comment by rupl** [comment]
So with your #38 merged, are we good on this one in your opinion? My local test with your modification was 👍

**Review by orakili — CHANGES_REQUESTED** [review]
Sorry, I still think the chevron should be an icon rather than a character.

VoiceOver on safari reads the character: "comma" (I know...) "the link title":

Screen Shot 2020-10-22 at 18 00 19

and the appearance is font-dependent. Ex: with `courier`:

Screen Shot 2020-10-22 at 18 05 50

So I think we should override the `gho-further-reading-link-formatter.html.twig` in the `common_design_subtheme` and add the following to the markup:

```
<svg class="cd-icon cd-icon--arrow-right" width="16" height="16" aria-hidden="true" focusable="false">
  <use xlink:href="#cd-icon--arrow-right"></use>
</svg>
```

**Comment by rupl** [comment]
Shucks. I was always under the impression that pseudos don't get read by VO or other screen readers, hence why it's not good to put important content in there (e.g. the `source` attribute from previous versions of this PR). But hey if the reading is awkward I can't argue. SVG it is.

**Review by orakili — APPROVED** [review]
Works well, thanks.

**Pull request merged**